modules/core/test/v2/unit/keychains.ts

```diff
@@ -40,6 +42,41 @@ describe('V2 Keychains', function v2keychains() {
     }));
   });
 
+  /**
+   * This section's intention is to provide some key generation sanity checking. generateKeyPair is a general surface
+   * for key generation but the keys are treated the same by BitGo down the line. Any SECP256K1 based coins key-pairs can
+   * be re-used so need to be the same.
+   **/
+  describe('Key generation enforcement for SECP256K1', function() {
```

Comment:
I don't quite follow this. What are we enforcing exactly? This test just makes sure each coin can take a seed and generate the expected xpub.

Reply:
I commented in private, it basically has to do with how we do key gen here